pages/clubs-toolkit.jsx

```diff
@@ -26,8 +28,6 @@ var Toolkit = React.createClass({
   render: function() {
     return (
       <div>
-        <p>{this.props.header}</p>
```

Comment:
ummm did you mean we don't need to a React component for `Toolkit`?  I think for readability reason I would prefer keeping it as a [component](https://github.com/mmmavis/teach.webmaker.org/blob/63ba880ede456a5f158ef92fbfc498a82a4f8779/pages/clubs-toolkit.jsx#L27-L35).